### DISCUSSION TITLE
Should you open donations for this project?

### DISCUSSION THREAD

**Discussion by davidcollini** [snuffyDev/Beatbump#46] [General]
There's a lot that donations can help with for beatbump, mainly implementing a proxy, sponsorblock support, and project speed in general. Opening a few crypto wallets, verifying on brave and opening a liberapay account should be great ways of funding this project, continue the great work!

**Comment by snuffyDev** [comment]
I've been thinking about it, since Beatbump is just now getting close to the free tier limits for Cloudflare Workers and I don't have a super steady source of income right now. I'm in the process now of setting up GitHub Sponsors so that way people can easily and directly donate and easily receive GitHub-specific perks. I'm soon gonna be finished setting that up, just need to do the tax paperwork as well as fully flesh out the donation tiers
